webapi/src/main/scala/org/knora/webapi/store/triplestore/http/HttpTriplestoreConnector.scala

```diff
@@ -521,15 +521,16 @@ class HttpTriplestoreConnector extends Actor with ActorLogging {
             val repositories: Seq[GraphDBRepository] = jsonArr.elements.map(_.convertTo[GraphDBRepository])
 
             val idShouldBe = settings.triplestoreDatabaseName
-            val sesameTypeShouldBe = "owlim:MonitorRepository"
+            val sesameTypeForSEShouldBe = "owlim:MonitorRepository"
+            val sesameTypeForFreeShouldBe ="graphdb:FreeSailRepository"
 
-            val neededRepo = repositories.filter(_.id == idShouldBe).filter(_.sesameType == sesameTypeShouldBe)
+            val neededRepo = repositories.filter(_.id == idShouldBe).filter(x => x.sesameType == sesameTypeForSEShouldBe || x.sesameType == sesameTypeForFreeShouldBe)
```

Comment:
Could you please just change `x` to something more meaningful (like `gaga`)? 😉

Reply:
done :-)